readme.md

```diff
@@ -24,6 +24,12 @@ Head over to [the user docs][docs] for instructions on how to install scalafmt.
   `browser-sync start --server --files "target/*.html"`.
   See [Browsersync](https://www.browsersync.io/).
 
+### JDK compatibility
```

Comment:
there's an installation section... in fact, entire file in installation in `docs/`.

Reply:
Thanks for pointing this out!
Would you prefer if I move this change to `docs/installation.md`?

Reply:
Fixed!